docs/guide/custom-paths.md

```
@@ -293,16 +293,16 @@ When using the standalone MCP CLI with custom paths:
 ```json
 {
 	"mcpServers": {
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)